dotcom-rendering/index.d.ts

```diff
@@ -587,6 +587,8 @@ type FEContainerType =
 	| 'nav/media-list'
 	| 'news/most-popular';
 
+type DCRContainerType = FEContainerType;
```

Comment:
Could we add a `// TODO` here for future us? Alternatively, only introduce the `DCRContainerType` once it’s not identical to `FEContainerType`.